astro/src/content/docs/lifecycle/authenticate-users/oauth/index.mdx

```diff
@@ -5,6 +5,7 @@ navcategory: login-methods
 section: lifecycle
 subcategory: authenticate users
 tertcategory: oauth
+topOfNav: true
```

Comment:
This page is an overview but is not marked as such in the nav.
![Screenshot 2023-11-17 at 11 30 35 AM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/afc2eaef-aec6-4f8c-8512-30984c4dea21)